README.md

```diff
@@ -93,5 +94,111 @@ When you have any questions, find a bug or have an idea for a feature you can cr
 
 ---
 
-Feel free to reach out, report issues, or share your ideas by creating an issue here on GitHub.  
+Feel free to reach out, report issues, or share your ideas by creating an issue [here](https://github.com/Dlurak/dlool_frontend_v2/issues) on GitHub.  
 Enjoy using Dlool!
+
```

Comment:
Awesome that you want to add German Translations! But I feel like the README.md should only be in english how about creating a README.de.md. Feel free to suggest other names.